src/utils/default-config.ts

```diff
@@ -0,0 +1,42 @@
+import type { BootstrapInit } from '@libp2p/bootstrap'
+import type { AddressManagerInit, ConnectionManagerInit } from 'libp2p'
+
+export interface BootstrapConfig {
+  addresses: Omit<AddressManagerInit, 'announceFilter'>
+  connectionManager: ConnectionManagerInit
+  bootstrap: BootstrapInit
+  identity: {
+    peerId: string
+    privKey: string
+  }
+}
+
+export const defaultConfig: BootstrapConfig = {
+  addresses: {
+    announce: [],
+    noAnnounce: [],
+    listen: [
+      '/ip4/0.0.0.0/tcp/4003/ws',
+      '/ip4/0.0.0.0/tcp/4001',
+      '/ip6/::/tcp/4004/ws',
+      '/ip6/::/tcp/4001',
+      '/ip4/0.0.0.0/udp/4002/webrtc'
+    ]
+  },
+  bootstrap: {
+    list: [
+      '/dns4/am6.bootstrap.libp2p.io/tcp/443/wss/p2p/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb',
```

Comment:
I would add the full list from 

https://github.com/ipfs/helia/pull/649/files#diff-1b64c902120625f3cd192eb7b8f10b323c4952735118b52ed87098ab4f7a6526R4-R10

Reply:
addressed in https://github.com/libp2p/js-libp2p-amino-dht-bootstrapper/pull/201